docs/UserGuide.md

```diff
@@ -145,6 +145,22 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Adding an order under a Person : `addOrder`
+
+Adds an order under a specified person from the address book.
+
+Format: `addOrder INDEX i/ITEM d/DATE [c/QUANTITY]`
+
+* List all orders of the person at the specified `INDEX`.
```

Comment:
Is this correct? I think it should be add an order under

Reply:
fixed